journalpump/journalpump.py

```diff
@@ -1094,6 +1183,7 @@ def _get_or_generate_json(self, field_filter, extra_field_values, data):
             timestamp = datetime.datetime.utcfromtimestamp(data["REALTIME_TIMESTAMP"])
         else:
             timestamp = datetime.datetime.utcnow()
+            data["REALTIME_TIMESTAMP"] = timestamp.timestamp()
```

Comment:
Don't think this needs to be added.

Reply:
I agree that it is not pretty, but I'm using that in the sender since the `data["timestamp"]` contains a string formatted date and CloudWatch wants timestamp as Epoch. I don't want to parse Epoch from string timestamp so I added the `REALTIME_TIMESTAMP` to the data here. I'm not entirely sure if journald adds this to every log entry, but since there is that `if "REALTIME_TIMESTAMP" in data:` I assumed that it is not always there.